src/aircraft/FlyByWireA380X.ts

```diff
@@ -0,0 +1,141 @@
+        },
+        [AircraftFeature.TaxiLights]: {
+            LIGHTING_LANDING_1: FeatureType.Int,
+        },
+        [AircraftFeature.WingLights]: {
+            LIGHTING_WING_0: FeatureType.Int,
+        },
+        [AircraftFeature.APU]: {
+            A32NX_OVHD_APU_MASTER_SW_PB_IS_ON: FeatureType.Int,
+            A32NX_OVHD_APU_START_SW_PB_IS_ON: FeatureType.Int,
+        },
+        [AircraftFeature.Battery]: {
+            A32NX_OVHD_ELEC_BAT_1_PB_IS_AUTO: FeatureType.Int,
+            A32NX_OVHD_ELEC_BAT_2_PB_IS_AUTO: FeatureType.Int,
+            A32NX_OVHD_ELEC_BAT_ESS_PB_IS_AUTO: FeatureType.Int,
+            A32NX_OVHD_ELEC_BAT_APU_PB_IS_AUTO: FeatureType.Int,
+        },
+        [AircraftFeature.Seatbelts]: {
+            XMLVAR_SWITCH_OVHD_INTLT_SEATBELT_Position: FeatureType.Int,
+            A32NX_FLAPS_HANDLE_INDEX: FeatureType.Int,
+            A32NX_GEAR_HANDLE_POSITION: FeatureType.Int,
+        },
+        [AircraftFeature.EmergencyLights]: {
+            XMLVAR_SWITCH_OVHD_INTLT_EMEREXIT_Position: FeatureType.Int,
+        },
+        [AircraftFeature.AntiIce]: {
+            XMLVAR_Momentary_PUSH_OVHD_ANTIICE_ENG1_Pressed: FeatureType.Int,
+            XMLVAR_Momentary_PUSH_OVHD_ANTIICE_ENG2_Pressed: FeatureType.Int,
+            XMLVAR_Momentary_PUSH_OVHD_ANTIICE_ENG3_Pressed: FeatureType.Int,
+            XMLVAR_Momentary_PUSH_OVHD_ANTIICE_ENG4_Pressed: FeatureType.Int,
+            XMLVAR_Momentary_PUSH_OVHD_ANTIICE_WING_Pressed: FeatureType.Int,
+        }
+    }
+
+    flapNames: FlapNames = {
+        0: 'UP',
+        1: 'CONF 1',
+        2: 'CONF 1+F',
+        3: 'CONF 2',
+        4: 'CONF 3',
+        5: 'FULL',
+    }
+
+    match(title: string, icao: string, config_path: string): boolean {
+        if (!title.includes('a380')) {
+            return false;
+        }
+
+        return ['fbw', 'flybywire'].some((w) => title.includes(w));
+    }
+
+    beaconLights(value: number): FeatureState {
+        return value === 1;
+    }
+
+    landingLights(value: number): FeatureState {
+        return value === 1;
+    }
+
+    logoLights(value: number): FeatureState {
+        return value === 1;
+    }
+
+    navigationLights(value: number): FeatureState {
+        return value === 1;
+    }
+
+    strobeLights(value: number): FeatureState {
+        if (value === 1) {
+            return null;
+        }
+        return value === 0;
+    }
+
+    wingLights(value: number): FeatureState {
+        return value === 1;
+    }
+
+    taxiLights(value: number): FeatureState {
+        return value === 0 || value === 1;
+    }
+
+    APU(mastersw: number, startsw: number): FeatureState {
+        return mastersw === 1 && startsw === 1;
+    }
+
+    battery(bat1: number, bat2: number, ess: number, apu: number): FeatureState {
+        return bat1 === 1 || bat2 === 1 || ess === 1 || apu === 1
+    }
+
+    emergencyLights(value: number): FeatureState {
+        return value !== 2;
```

Comment:
I assume 2 is `OFF`, 1 is `AUTO` and 0 is `ON` which is kind of strange to define, but this is FBW so I expect anything :)

Reply:
Exactly, that's it